lighthouse-core/test/report/html/renderer/performance-category-renderer-test.js

```diff
@@ -288,14 +288,23 @@ describe('PerfCategoryRenderer', () => {
       }
     });
 
-    it('uses null if the metric is missing its value', () => {
+    it('uses null if the metric\'s value is undefined', () => {
       const categoryClone = JSON.parse(JSON.stringify(category));
       const lcp = categoryClone.auditRefs.find(audit => audit.id === 'largest-contentful-paint');
       lcp.result.numericValue = undefined;
       lcp.result.score = null;
       const href = renderer._getScoringCalculatorHref(categoryClone.auditRefs);
       expect(href).toContain('largest-contentful-paint=null');
     });
+
+    it('uses null if the metric\'s value is null (LR)', () => {
+      const categoryClone = JSON.parse(JSON.stringify(category));
+      const lcp = categoryClone.auditRefs.find(audit => audit.id === 'largest-contentful-paint');
+      lcp.result.numericValue = null;  // In LR, undefined becomes null
```

Comment:
```suggestion
      lcp.result.numericValue = null; // In LR, undefined becomes null
```

Reply:
the comment is still wrong even without the extra space, though :D

Reply:
I thought the status was it *should* be wrong, but for some reason we're seeing it be true?

Reply:
I think it was that only Paul (possibly?) saw it, and only in the non-real deployment

Reply:
we think it *should* be impossible, but this is a better way to handle this situation even if it's not possible, so we're proceeding here.